docs/dev/new_check_howto.md

```diff
@@ -571,10 +571,17 @@ sudo datadog-agent integration install -w /path/to/wheel.whl
 ```
 
 **Windows** (Ensure that your shell session has _administrator_ privileges):
+
+For agent versions <= 6.12:
```

Comment:
Should this be `For agent versions <= 6.11:` ? Since in agent 6.12 it's already `embedded2`.

Reply:
reverted